rest-api-spec/src/main/resources/rest-api-spec/test/search/260_sort_mixed.yml

```diff
@@ -119,3 +120,85 @@
   - match: { status: 400 }
   - match: { error.type: search_phase_execution_exception }
   - match: { error.caused_by.reason: "Can't do sort across indices, as a field has [unsigned_long] type in one index, and different type in another index!" }
+
+---
+"search across indices with mixed long and double and float numeric types":
+  - skip:
+      version: " - 2.10.99"
+      reason: half float was broken before 2.11
```

Comment:
2.11 we are releasing a patch. The fix was part of that patch.

Reply:
Technically, 2.11.0 is already released, I'm not sure if this meets the criteria to go as patch in 2.11.1 as it is not critical (disabling half_float optimization is, which is already checked in).

Reply:
https://opensearch.slack.com/archives/C0561HRK961/p1698428388137519

Not for this commit `disabling half_float optimization is, which is already checked in` but it is going with another critical commits for which we are releasing patch. It is fine to include in that case for 2.11.1 right ?

Reply:
It already got in though.
https://github.com/opensearch-project/OpenSearch/pull/11005

Reply:
Shouldn't this be 2.11.0 where it is still broken?

Reply:
For this test framework, it always creates opensearch test cluster from latest patched version, so from 2.11 version family, it will be always 2.11.1. That is the reason this test is passing....

Reply:
@gashutos I think we won't make it to 2.11.1 but 2.12

Reply:
@reta @backslasht `half_float` type is supported with PR https://github.com/opensearch-project/OpenSearch/pull/10999. Which is included in 2.11.1 patch. With that, the tests writtten here should pass for 2.11. 
This PR we can make it 2.12 which is the optimizing `half_float` sort. So we should be good here for specifying. 2.10.99 istead 2.11 as the test is expected to pass in version 2.11

Reply:
Got it, thanks for clarifying @gashutos